### DISCUSSION TITLE
Mantine React Table Icons background color issues

### DISCUSSION THREAD

**Discussion by HxnDev** [KevinVandy/mantine-react-table#444] [Q&A]
### mantine-react-table version

v2.0.0-beta.5

### react & react-dom versions

v18.3.1

### Describe the bug and the steps to reproduce it

I am using Mantine React Tables and everything is good. However, I enabled the sorting and filtering options of my table and then the sort icon (arrows) starting showing up with the green background color (which is the products primary color). I keep trying to set the background as transparent, but nothing works.

<img width="1734" alt="image" src="https://github.com/user-attachments/assets/cf053fb1-1554-4b15-91cb-e23993b05f01">

The inspect element shows the following:

<img width="580" alt="image" src="https://github.com/user-attachments/assets/bdbd5c70-7b98-442f-862a-531d3aaa1847">

<button class="mantine-focus-auto mantine-active mrt-table-head-sort-button MRT_TableHeadCellSortLabel-module_sort-icon__zs1xA m_8d3f4000 mantine-ActionIcon-root m_87cf2631 mantine-UnstyledButton-root" type="button" aria-label="Sort by Country ascending" style="--ai-color: var(--mantine-color-white);"><span class="m_8d3afb97 mantine-ActionIcon-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="tabler-icon tabler-icon-arrows-sort "><path d="M3 9l4 -4l4 4m-4 -4v14"></path><path d="M21 15l-4 4l-4 -4m4 4v-14"></path></svg></span></button>

-> On the element.style window in inspect. i can see the primary color being assigned to the background, but if i put the same classname (.m_8d3f4000) in code to remove background, it does nothing.

<img width="492" alt="image" src="https://github.com/user-attachments/assets/1c6d540e-244b-495c-9abb-e6e90ce0ff66">

### Minimal, Reproducible Example - (Optional, but Recommended)

```typescript

const Table = () => {
  const table = useMantineReactTable({
    columns,
    data: rowData,
    enableSorting: true,
  })
  return table
}
```

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by alessandrojcm** [comment]
Hi, I believe this was fixed, can you try the latest beta and confirm please?

**Comment by HxnDev** [comment]
> Hi, I believe this was fixed, can you try the latest beta and confirm please?

Hello @alessandrojcm 

I updated to version 2-beta7, but the error still persists:

![image](https://github.com/user-attachments/assets/a952b5fe-6500-4d95-acaa-b69b6298bf2a)


![image](https://github.com/user-attachments/assets/42d7e745-db7f-4d2c-93c4-1223fd8a2e55)


```
const columns = useMemo<MRT_ColumnDef<ContributionItems>[]>(
    () => [
      {
        accessorKey: 'project.name',
        header: 'Project Name',
        Cell: ({ cell }) => {
          const project_name = cell.getValue<string>() || 'N/A'
          return <TruncatedTextWithTooltip text={project_name} />
        },
      },
      {
        accessorKey: 'project.location.countryName',
        header: 'Country',
        Cell: ({ cell }) => {
          const country = cell.getValue<string>() || 'N/A'
          return <TruncatedTextWithTooltip text={country} />
        },
      },
```

```
const table = useMantineReactTable({
    columns,
    data: rowData,
    rowCount: totalRowCount,
    pageCount: Math.ceil(totalRowCount / pagination.pageSize),
    enablePagination: false,
    enableGlobalFilter: false,
    mantineToolbarAlertBannerProps: error
      ? {
          color: 'red',
          children: error.message,
        }
      : undefined,
    state: {
      isLoading: loading,
      showAlertBanner: !!error,
      showSkeletons: false,
      pagination,
    },
    onPaginationChange: (newPagination) => {
      if (typeof newPagination === 'function') {
        setPagination((prevPagination) => newPagination(prevPagination))
      } else {
        setPagination(newPagination)
      }
    },
  })
```

**Comment by chindathorn** [comment]
@HxnDev make sure you add `import 'mantine-react-table/styles.css';` in your **root file**

**Comment by HxnDev** [comment]
> @HxnDev make sure you add `import 'mantine-react-table/styles.css';` in your **root file**

Hello @chindathorn 
Thank you. I did try that and it works.

However just a followup question, now the table toolbar is moved to the far right side and theres this empty space in the bottom. Is there anything i could do about it?

Picture 1: Previous Table version

![image](https://github.com/user-attachments/assets/1f60caf5-6b10-40d1-8658-5146ffecc0a3)

Picture 2: Current Table after adding style.css

![image](https://github.com/user-attachments/assets/a8a3b848-da26-4540-98d5-8c93320ae72d)

**Comment by chindathorn** [comment]
@HxnDev try this
```typescript
const table = useMantineReactTable({
    mantineTopToolbarProps: {
        style: { justifyContent: 'start' },
        // ...your other custom styles
    },
})
```

**Comment by HxnDev** [comment]
Hey @chindathorn 

I did try that, but no use

```
const table = useMantineReactTable({
    columns,
    data: rowData,
    rowCount: totalRowCount,
    pageCount: Math.ceil(totalRowCount / pagination.pageSize),
    enablePagination: false,
    enableGlobalFilter: false,
    manualFiltering: true,
    manualSorting: true,
    manualPagination: true,
    enableColumnActions: true,
    mantineTopToolbarProps: {
      style: { justifyContent: 'start' },
      // ...your other custom styles
    },
    mantineToolbarAlertBannerProps: error
      ? {
          color: 'red',
          children: error.message,
        }
      : undefined,
    state: {
      isLoading: loading,
      showAlertBanner: !!error,
      showSkeletons: false,
      pagination,
      sorting,
      columnFilters,
      columnVisibility,
    },
```

I also tried the below too but still no solution:

```
const table = useMantineReactTable({
    columns,
    data: rowData,
    rowCount: totalRowCount,
    pageCount: Math.ceil(totalRowCount / pagination.pageSize),
    enablePagination: false,
    enableGlobalFilter: false,
    manualFiltering: true,
    manualSorting: true,
    manualPagination: true,
    enableColumnActions: true,
    layoutMode: 'semantic',
mantineTableHeadCellProps: {
      style: {
        textAlign: 'left',
      }
    },
```

<img width="1744" alt="image" src="https://github.com/user-attachments/assets/71c9bd81-624d-49d6-972f-88d195eae1d8">

**Reply by hillac** [reply]
Did you figure out how to get it on the left?
